pkg/preparer/orchestrate.go

```diff
@@ -165,6 +169,27 @@ func (p *Preparer) handlePods(podChan <-chan pods.PodManifest, quit <-chan struc
 	}
 }
 
+// check if a manifest satisfies the signature requirement of this preparer
+func (p *Preparer) verifySignature(manifest pods.PodManifest, logger logging.Logger) bool {
+	// do not remove the logger argument, it's not the same as p.Logger
+	if p.keyring == nil {
```

Comment:
If there is no keyring, then the preparer is operating in unauthenticated mode, and the signature of the manifest (if any) is irrelevant. Therefore the manifest should be accepted.